[esx]/es_extended/server/main.lua

```diff
@@ -360,6 +360,11 @@ if not Config.OxInventory then
 		local playerId = source
 		local sourceXPlayer = ESX.GetPlayerFromId(playerId)
 		local targetXPlayer = ESX.GetPlayerFromId(target)
+        local distance = #(GetEntityCoords(GetPlayerPed(playerId)) - GetEntityCoords(GetPlayerPed(target)))
+        if not sourceXPlayer then return end
+        if not targetXPlayer then print("Cheating: " ..  GetPlayerName(playerId)) return end
+        if distance > 4.0 then print("Cheating: " ..  GetPlayerName(playerId)) return end
```

Comment:
Make a config for the distance check since is not everyone that wanna have only `4.0`. Also, since this is a public project, is not everyone that know how to change that :D